src/library/library.cpp

```diff
@@ -223,6 +223,26 @@ void Library::stopPendingTasks() {
     }
 }
 
+void Library::bindSearchboxWidget(WSearchLineEdit* pSearchboxWidget) {
+    connect(pSearchboxWidget,
+            &WSearchLineEdit::search,
+            this,
+            &Library::search);
+    connect(pSearchboxWidget,
+            &WSearchLineEdit::disableSearch,
+            this,
+            &Library::disableSearch);
+    connect(pSearchboxWidget,
+            &WSearchLineEdit::restoreSearch,
+            this,
+            &Library::restoreSearch);
+    connect(this,
+            &Library::setTrackTableFont,
+            pSearchboxWidget,
+            &WSearchLineEdit::slotSetFont);
+    emit(setTrackTableFont(m_trackTableFont));
```

Comment:
Please remove the braces following the `emit` keyword. Jan has just cleaned up the code base to fix those clazy warnings.